helper.py

```diff
@@ -1,15 +1,52 @@
 from functools import reduce
 
-from mappings import map
+from mappings import map, word_to_plus_code_mapping
 
 
-def two_to_word(code: str):
+sep = '.'
+
+
+def get_word(code: str):
     return map[code]
 
 
-def olc_to_phrase(olc: str):
-    code = olc[:-4]
-    # Split string into array of 2 char elements.
-    char_arr = [code[i: i+2] for i in range(0, len(code), 2)]
+def get_code(word: str):
+    return word_to_plus_code_mapping[word]
+
+
+def single_char_phrase(olc: str):
+    code = olc.replace('+', '')
+    char_arr = [code[i: i+1] for i in range(0, len(code))]
+    return reduce(lambda acc, s: acc + sep + get_word(s), char_arr, '')[1:]
+
+
+def double_char_phrase(olc: str):
+    char_arr = []
+    code = olc.replace('+', '')
+    if len(code) % 2 == 0:
+        # Split string into array of 2 char elements.
+        char_arr = [code[i: i+2] for i in range(0, len(code), 2)]
```

Comment:
`char_arr` is the same result regardless if it's even or not using this function. we can just use this line on its own